checkov/cdk/checks/python/CloudFrontTLS12.yaml

```diff
@@ -0,0 +1,13 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_174
+  name: Verify CloudFront Distribution Viewer Certificate is using TLS v1.2
+  category: ENCRYPTION
+scope:
+  languages:
+    - python
+definition:
+  pattern: aws_cdk.aws_cloudfront.CfnDistribution(<ANY>)
+  conditions:
+    - not_pattern: aws_cdk.aws_cloudfront.CfnDistribution(<ANY>, distribution_config=aws_cdk.aws_cloudfront.CfnDistribution.DistributionConfigProperty(<ANY>, viewer_certificate=aws_cdk.aws_cloudfront.CfnDistribution.ViewerCertificateProperty(<ANY>, minimum_protocol_version='TLSv1.2' ,<ANY>) ,<ANY>) ,<ANY>)
```

Comment:
Same comments as above (regarding the permutations issue)

Reply:
in this case it is nested the order cannot change